.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```

Comment:
Let's make sure we follow the [Microverse linters repo for javascript](https://github.com/microverseinc/linters-config/tree/master/javascript) instructions to set up our linters properly, tend to that, lint whatever issues that may arise from the proper configuration of eslint and stickler, and you should be golden.